client/components/AddToCart.jsx

```diff
@@ -10,47 +10,91 @@ import imgBell from './assets/etsy_bell.png';
 
 const Button = styled.button`
   background: black;
+  font-family: "Graphik Webfont",-apple-system,BlinkMacSystemFont,"Roboto","Droid Sans","Segoe UI","Helvetica",Arial,sans-serif;
+  font-size: 15px;
+  font-stretch: 100%;
+  font-weight: 700;
+  line-height: 18px;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  border-bottom-left-radius: 3px;
+  width: 282px;
   color: white;
   width: 200px;
+  box-sixing: border-box;
```

Comment:
Is this supposed to be
```suggestion
  box-sizing: border-box;
```
?